src/test/resources/last-node-to-leave-jboss-ejb-client.properties

```diff
@@ -0,0 +1,48 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2010 Red Hat, Inc., and individual contributors
+# as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+remote.connectionprovider.create.options.org.xnio.Options.SSL_ENABLED=false
+
+remote.connections=one,two
+
+# connection to a node at protocol://host:port
+remote.connection.one.host=localhost
+remote.connection.one.port=6999
+remote.connection.one.protocol=remote
+remote.connection.one.connect.options.org.xnio.Options.SASL_POLICY_NOANONYMOUS=false
+remote.connection.one.username=test
+remote.connection.one.password=test
+remote.connection.one.realm=default
+
+# connection to a node at protocol://host:port
+remote.connection.two.host=localhost
+remote.connection.two.port=7099
+remote.connection.two.protocol=remote
+remote.connection.two.connect.options.org.xnio.Options.SASL_POLICY_NOANONYMOUS=false
+remote.connection.two.username=test
+remote.connection.two.password=test
+remote.connection.two.realm=default
+
+# the nodes are clustered in a cluster called ejb and have names node1, node2
+remote.clusters=ejb
+remote.cluster.ejb.node.node1.username=test
+remote.cluster.ejb.node.node1.password=test
+# remote.cluster.ejb.clusternode.selector=org.jboss.ejb.client.test.LastNodeToLeaveTestCase$TestSelector
```

Comment:
Do we need commented lines?

Reply:
No we don't. Thanks for pointing that out.

Reply:
Done